x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.tsx

```diff
@@ -180,7 +186,7 @@ export const CspPolicyTemplateForm = memo<PackagePolicyReplaceDefineStepExtensio
     ];
 
     return (
-      <div>
+      <div data-test-subj={'test'}>
```

Comment:
please give more informative test subject names

Reply:
This was inadvertent addition. I will remove this code